gradle/wrapper/gradle-wrapper.properties

```
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-8.3-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.7-bin.zip
```

```suggestion
distributionUrl=https\://services.gradle.org/distributions/gradle-8.6-bin.zip
```

To be consistent with my other projects